### ISSUE TITLE
[Feature Request] Make  settings panel as another tab instead of a separate window.

### ISSUE THREAD

**Issue by nekojin116** [Windows-Apps-Hub/UnitedSets#26]
![image](https://user-images.githubusercontent.com/72446956/218558805-77b6215c-9191-4944-89be-c2009c8b4fa7.png)

I think making it separate will give more aesthetics to the app, like it, and good luck everyone!

**Comment by Get0457** [comment]
Good Idea. However, the settings window are not usually used often so this change might take a bit longer to implement because this will require a lot of backend code changes.

**Comment by GetGet99** [comment]
Low priority as it's unlikely going to be implemented soon

**Comment by nekojin116** [comment]
i forgot i even made this issue, glad to see this still maintained as i moved to linux and cant use it anymore
